resdk/resolwe.py

```diff
@@ -508,9 +509,15 @@ def get(self, uid):
 
         """
         if re.match('^[0-9]+$', str(uid)):  # iud is ID number:
-            return self.resource(id=uid, resolwe=self.resolwe)
+            if self.endpoint == 'presample':
+                return self.resource(id=uid, resolwe=self.resolwe, presample=True)
+            else:
+                return self.resource(id=uid, resolwe=self.resolwe)
         else:  # uid is slug
-            return self.resource(slug=uid, resolwe=self.resolwe)
+            if self.endpoint == 'presample':
+                return self.resource(slug=uid, resolwe=self.resolwe, presample=True)
+            else:
+                return self.resource(slug=uid, resolwe=self.resolwe)
```

Comment:
Write this like:

    sample_filter = {'id': uid} if uid.isdigit() else {'slug': uid}
    if self.endpoint == 'presample':
        sample_filter['presample'] = True
    return self.resource(resolwe=self.resolwe, **sample_filter)